api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -9,34 +9,65 @@
  */
 package org.openmrs.notification.db.hibernate;
 
+import org.junit.Assert;
+
 import org.junit.Before;
 import org.junit.Test;
+import org.openmrs.User;
 import org.openmrs.api.context.Context;
-import org.openmrs.notification.AlertService;
+import org.openmrs.notification.Alert;
 import org.openmrs.test.BaseContextSensitiveTest;
+import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * This class tests the hibernate alert data access. TODO Consider changing this and all subsequent
  * tests to use dbunit
  */
 public class HibernateAlertDAOTest extends BaseContextSensitiveTest {
 	
+	private static final String DATA_XML = "org/openmrs/api/db/hibernate/include/HibernateAlertDAOTestDataSet.xml";
+	
+	@Autowired
+	private HibernateAlertDAO hibernateAlertDAO;
+	
 	@Before
-	public void runBeforeEachTest() {
-		authenticate();
+	public void setUp() {
+		executeDataSet(DATA_XML);
+	}
+
+	@Test
+	public void saveAlert_shouldSaveAlertToDb() {
```

Comment:
@ibacher   hi ian made a fix for this method,thanks

Reply:
@ibacher   on squashing this, i get different commits that i never made using(git rebase -i master)

Reply:
@HerbertYiga That's kind of expected. Those commits are probably the result of changed committed to `master`. You don't have to worry about squashing things, though, as GitHub will handle that for us.